native/build.gradle

```diff
@@ -36,7 +36,8 @@ checkstyle {
     configProperties = ["suppressionFile" : file("${rootDir}/build-config/checkstyle/build/suppressions.xml")]
 }
 
-checkstyleMain.dependsOn(":build-config:checkstyle:downloadMultipleFiles")
+checkstyleMain.dependsOn ":checkstyle:downloadCheckstyleRuleFiles"
+checkstyleTest.dependsOn ":checkstyle:downloadCheckstyleRuleFiles"
```

Comment:
We don't need this since we don't have native tests in url module.

Reply:
Ah yes. Fixed with https://github.com/ballerina-platform/module-ballerina-url/pull/219/commits/4b5d53e1aba227d326b3b213f91892f8b636f973